# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1224616453 14400
#      Tue Oct 21 15:14:13 2008 -0400
# Node ID b12a2975bf7e1b32b25fd8ea9912f346076d90ec
# Parent  8067a06c901cd2e1fd7ad07569bd5145fc9b1819
newplot.m: delete stray debugging code

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,14 +1,16 @@
 2008-10-21  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/clf.m: Improve Matlab compatibility.
 
 2008-10-21  John W. Eaton  <jwe@octave.org>
 
+	* plot/newplot.m: Delete stray debugging code.
+
 	* plot/gnuplot_drawnow.m: Pass handle to __go_draw_figure__, not
 	struct.  Use get instead of examining struct fields directly.
 	* plot/__go_draw_figure__.m: First arg is now handle, not figure
 	object struct.  Use get instead of examining struct elements
 	directly.  Use allhild instead of looking at children field of
 	figure object struct.
 
 2008-10-20  Ben Abbott  <bpabbott@mac.com>
diff --git a/scripts/plot/newplot.m b/scripts/plot/newplot.m
--- a/scripts/plot/newplot.m
+++ b/scripts/plot/newplot.m
@@ -44,18 +44,13 @@ function newplot ()
       case "add"
       case "replacechildren"
       case "replace"
 	__go_axes_init__ (ca, "replace");
 	__request_drawnow__ ();
       otherwise
 	error ("newplot: unrecognized nextplot property for current axes");
     endswitch
-
-    yt = get (ca, "ylabel");
-    if (! strcmp (get (yt, "type"), "text"))
-      disp("BAD!!!!!");
-      get(yt)
-    endif
   else
     print_usage ();
   endif
+
 endfunction
